.github/workflows/check-pip-dependencies.yml

```diff
@@ -0,0 +1,33 @@
+name: Check pip dependencies
+
+on:
+  push:
+    paths:
+      - requirements**.txt
```

Comment:
Path filtering is done on the full path, so this should be `**/requirements*.txt`.